### DISCUSSION TITLE
"Single-use" config file

### DISCUSSION THREAD

**Discussion by bruce-swart** [SafeExamBrowser/seb-win-refactoring#294] [Q&A]
Distributing a SEB config file to students to start a Microsoft Forms Quiz on their personal machines. They get provided with the config password to open once they have SEB installed. Is there a way to make this "single-use" using SEB? Don't want to allow them to distribute the config file to others.

**Comment by dbuechel** [comment]
No, not with a single configuration file. You could e.g. distribute a generic configuration file which leads the students to a portal page where they then can download and reconfigure SEB with the actual exam configuration. Or you could use SEB-Server and manage and monitor your exams there.
